packages/ui/src/shared-style-variables.css

```
@@ -473,88 +466,88 @@
     --canary-ai-button-stop-3: var(--canary-purple-400);
     --canary-ai-button-stop-4: var(--canary-cyan-200);
 
-    --diff-grey: var(--canary-foreground-05);
-    --diff-add-content--: var(--canary-green-50);
-    --diff-del-content--: var(--canary-red-50);
-    --diff-add-lineNumber--: var(--canary-green-600);
-    --diff-del-lineNumber--: var(--canary-red-600);
-    --diff-plain-content--: var(--canary-background-01);
-    --diff-expand-content--: var(--canary-background-01);
-    --diff-plain-lineNumber--: var(--canary-background-01);
-    --diff-plain-lineNumber-color--: var(--canary-foreground-05);
-    --diff-hunk-content--: var(--canary-accent-50);
-    --diff-hunk-lineNumber--: var(--canary-accent-600);
-    --diff-add-content-highlight--: var(--canary-green-100);
-    --diff-del-content-highlight--: var(--canary-red-100);
-    --diff-add-widget--: var(--canary-background-03);
-    --diff-add-widget-color--: var(--canary-foreground-02);
-    --diff-empty-content--: var(--canary-background-01);
-    --diff-hunk-content-color--: var(--canary-foreground-02);
+    --diff-grey: hsl(var(--canary-base-chrome-50));
+    --diff-add-content--: hsl(var(--canary-green-50));
+    --diff-del-content--: hsl(var(--canary-red-50));
+    --diff-add-lineNumber--: hsl(var(--canary-green-100));
+    --diff-del-lineNumber--: hsl(var(--canary-red-100));
+    --diff-plain-content--: hsl(var(--canary-background-01));
+    --diff-expand-content--: hsl(var(--canary-background-01));
+    --diff-plain-lineNumber--: hsl(var(--canary-background-01));
+    --diff-plain-lineNumber-color--: hsl(var(--canary-base-chrome-850));
+    --diff-hunk-content--: hsl(var(--canary-accent-50));
+    --diff-hunk-lineNumber--: hsl(var(--canary-accent-100) / 0.7);
```

Ooopps, just noticed this. For alpha, you need to switch to `hsla`
```suggestion
    --diff-hunk-lineNumber--: hsla(var(--canary-accent-100) / 0.7);
```